package/plugin-gargoyle-stamgr/files/www/js/sta_manager.js

```diff
@@ -0,0 +1,419 @@
+/*
+ * This program is copyright © 2019 Michael Gray and is distributed under the terms of the GNU GPL
+ * version 2.0 with a special clarification/exception that permits adapting the program to
+ * configure proprietary "back end" software provided that all modifications to the web interface
+ * itself remain covered by the GPL.
+ * See http://gargoyle-router.com/faq.html#qfoss for more information
+ */
+
+var stamgrStr=new Object(); //part of i18n
+
+var encryptionList = [];
+encryptionList['none'] = 'Open';
+encryptionList['psk'] = 'WPA-Personal';
+encryptionList['psk2'] = 'WPA2-Personal';
+encryptionList['sae'] = 'WPA3-Personal';
```

Comment:
On basic page we are using WPA PSK/WPA2 PSK, not WPA-Personal/WPA2-Personal. It would be good if we stick to uniform names

Reply:
Yes good point. I'll also add check code to disable WPA3 where it isn't available.